app/Mcp/Tools/GetInfrastructureOverview.php

```
@@ -72,16 +83,61 @@ public function handle(Request $request): Response
         }
 
+        $serverIds = $servers->pluck('id');
+        $openDeployments = $serverIds->isEmpty()
+            ? 0
+            : ApplicationDeploymentQueue::query()
+                ->whereIn('server_id', $serverIds)
+                ->whereIn('status', ['in_progress', 'queued'])
+                ->count();
+
+        // Count-only health hints (no full model hydration for apps).
+        $appNotRunningQuery = Application::ownedByCurrentTeamAPI($teamId);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Models\Application' in method 'handle'. (undefined)

(StaticAccess)